pkgs/applications/audio/pulseaudio-dlna/default.nix

```diff
@@ -0,0 +1,25 @@
+{ fetchFromGitHub, stdenv, pythonPackages, lame, opusTools, faac, flac, sox, vorbisTools }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "pulseaudio-dlna";
+  version = "2017-11-01";
```

Comment:
Why not 0.5.2? If you want a development snapshot, you can simply override the version.

Reply:
I did the latest as the author has not released in a long time.

Reply:
As project uses a couple python libraries this seems appropriate to not roadblock other projects, when updating python libraries.